src/components/ListItems/post-list.js

```diff
@@ -16,25 +16,29 @@ function PostList({ user_list }) {
 
   function checkLiked(item) {
     for (var i = 0; i < liked.length; i++) {
-      if (liked[i].object === item.id) {
```

Comment:
Do we not need triple = ?

Reply:
true. I was testing around and forget to redo this